src/ip/Intercept.h

```diff
@@ -48,7 +48,7 @@ class Intercept
      \retval 0  Full transparency is disabled.
      \retval 1  Full transparency is enabled and active.
      */
-    inline int TransparentActive() { return transparentActive_; };
+    int transparentActive() const { return transparentActive_ && tproxyEnabled(); };
```

Comment:
If (and only if) this code change survives, then please remove the extra trailing `;` and convert to bool, adjusting the documentation to say something like `/// whether full transparency was enabled at ./configure time and then passed runtime checks` (adjust as needed, of course).